### DISCUSSION TITLE
MSI BIOS FLASH ERROR

### DISCUSSION THREAD

**Discussion by itasli** [terminatorul/NvStrapsReBar#31] [Q&A]
Hey there @terminatorul , i just patched the latest bios image with the .ffs file, when I try to flash it one the bios i get "Failed, Wrong Image format 11!" anyway to fix/bypass that ?

**Comment by terminatorul** [comment]
Hi ! All I know about this are the instructions in [ReBarUEFI](https://github.com/xCuri0/ReBarUEFI/) project, see:

- https://github.com/xCuri0/ReBarUEFI/wiki/Flashing-modified-UEFI
- https://winraid.level1techs.com/t/guide-how-to-flash-a-modded-ami-uefi-bios/30627

Does your board have BIOS Flashback ?
Can you try the flashrom tool ?

**Reply by itasli** [reply]
Hey there, nope I don't have BIOS Flashback... will give a try to flashrom

**Reply by WolfJMZ** [reply]
Hey I'm having this same exact issue - did the flashrom tool work?

**Reply by itasli** [reply]
Hey @jammerxd, I didn't find any tutorial on how to use it actually so nope didn't tried...

**Reply by WolfJMZ** [reply]
Do you have the ability to try using the AMI tools? As pointed out in the 2nd link? (I'm not an expert either and am hoping someone else can chime in)
